angular-spa-ui/js/search-app/tags-module/tags-list-directive/tags-list-ctrl.js

```diff
@@ -0,0 +1,215 @@
+'use strict';
+
+module.exports = function($scope, tagService) {
+
+    // Property of tag list
+    $scope.tags = [];
+
+    // Property of create tag directive
+    $scope.form = {
+        tagName: ''
+    };
+
+    $scope.msgWorning = '';
+
+    // Define scope directive
+    //-------------------------
+
+    $scope.getCurrentScopeParam = function() {
```

Comment:
the second: I think it's a not so good approach for switching something, better to have passed options which can say where you use something